src/pages/home/report/ReportActionItem.js

```diff
@@ -80,15 +103,33 @@ class ReportActionItem extends Component {
     }
 
     render() {
+        const message = this.props.action.actionName === 'IOU'
+            ? (
+                <ReportActionItemIOUPreview
+                    report={this.props.report}
+                    action={this.props.action}
+                    isMostRecentIOUReportAction={this.props.isMostRecentIOUReportAction}
+                />
+            )
+            : <ReportActionItemMessage action={this.props.action} />;
         return (
             <PressableWithSecondaryInteraction onSecondaryInteraction={this.showPopover}>
                 <Hoverable>
                     {hovered => (
                         <View>
                             <View style={getReportActionItemStyle(hovered)}>
                                 {!this.props.displayAsGroup
-                                    ? <ReportActionItemSingle action={this.props.action} />
-                                    : <ReportActionItemGrouped action={this.props.action} />}
+                                    ? (
+                                        <ReportActionItemSingle
+                                            action={this.props.action}
+                                            message={message}
+                                        />
+                                    )
+                                    : (
+                                        <ReportActionItemGrouped
+                                            message={message}
+                                        />
```

Comment:
Is action missing from `ReportActionItemGrouped`, or is action unnecessary in `ReportActionItemSingle`?

Reply:
`action` is used for other purposes in `ReportActionItemSingle`. (It is used to show avatar of action's actor.)